coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorGrpcService.java

```diff
@@ -206,7 +207,10 @@ public void accessCluster(AccessClusterRequest request, StreamObserver<AccessClu
     AccessClusterResponse response;
     AccessManager accessManager = coordinatorServer.getAccessManager();
 
-    AccessInfo accessInfo = new AccessInfo(request.getAccessId(), Sets.newHashSet(request.getTagsList()));
+    Map<String, String> extraProperties = request.getExtraPropertiesMap();
+    AccessInfo accessInfo =
+            new AccessInfo(request.getAccessId(), Sets.newHashSet(request.getTagsList()),
+                    extraProperties == null ? Collections.emptyMap() : extraProperties);
```

Comment:
Why don't we use method `hasExtraPropertiesMap()`? If request' don't have member, it will give it a default value, the default value may not be null.

Reply:
Proto2 has this method but proto3 don’t have this.

Reply:
OK